app/models/user.rb

```diff
@@ -36,4 +40,10 @@ def valid_email
 
     errors.add :email, 'invalid'
   end
+
+  def remove_all_roles
```

Comment:
this callback has one argument sent in to it (`roles`)

Reply:
But it doesn't use the argument. So It doesn't need to be declared.